images/win/scripts/Installers/Install-Rust.ps1

```diff
@@ -11,13 +11,10 @@ $env:CARGO_HOME="C:\Rust\.cargo"
 
 # Download the latest rustup-init.exe for Windows x64
 # See https://rustup.rs/#
-Invoke-WebRequest -UseBasicParsing -Uri "https://win.rustup.rs/x86_64" -OutFile rustup-init.exe
+$rustupPath = Start-DownloadWithRetry -Url "https://win.rustup.rs/x86_64" -Name "rustup-init.exe"
 
 # Install Rust by running rustup-init.exe (disabling the confirmation prompt with -y)
-.\rustup-init.exe -y --default-toolchain=stable --profile=minimal
-
-# Delete rustup-init.exe when it's no longer needed
-Remove-Item -Path .\rustup-init.exe
+& $rustupPath -y --default-toolchain=stable --profile=minimal
```

Comment:
I see that previously Rustup was removed after installation, but in your changes no such line `Remove-Item -Path .\rustup-init.exe` - is it expected?

Reply:
This is expected. There is no need to manually remove `rustup-init.exe` because files that was downloaded by `Start-DownloadWithRetry` without `DownloadPath` argument is downloaded into `$env:TEMP` folder, that is automatically cleaned by `Finalize-VM` provisioner.

Reply:
got it, thanks!